src/app/cli/src/coda_worker_testnet.ml

```diff
@@ -149,10 +149,22 @@ let start_prefix_check logger workers events testnet ~acceptable_delay =
       !"lengths: %{sexp: int list} shared_prefix: %{sexp: string option} \
         shared_prefix_age: %d"
       lengths newest_shared shared_prefix_age ;
-    if not (shared_prefix_age <= 5) then (
-      Logger.fatal logger ~module_:__MODULE__ ~location:__LOC__
-        "prefix too old" ;
-      ignore (exit 1) ) ;
+    let chains_list = Array.to_list chains in
```

Comment:
this stuff is supposed to be in a different commit right?

Reply:
no, this is just adding metadata to the log.

Reply:
oh I see. i got confused since you were modifying the shared prefix test as part of other work